### ISSUE TITLE
`invert_colors` fails for some image formats

### ISSUE THREAD

**Issue by zzril** [Safe-DS/Stdlib#421]
### Describe the bug

The `invert_colors` method for images internally uses `ImageOps.invert(...)`:
[src/safeds/data/image/containers/_image.py](https://github.com/Safe-DS/Stdlib/blob/e7389669d76f9ad199bafdf617a6f9be01e0880b/src/safeds/data/image/containers/_image.py#L374-L385):
```py
    def invert_colors(self) -> Image:
        """
        Return the image with inverted colors.


        Returns
        -------
        result : Image
            The image with inverted colors.
        """
        image_copy = copy.deepcopy(self)
        image_copy._image = ImageOps.invert(image_copy._image)
        return image_copy
```

This `ImageOps.invert(...)` method can fail with an obscure `OSError` for certain, non-RGB image formats.
For example, it fails for [this](https://github.com/Safe-DS/Stdlib/pull/419/commits/a3c1b1d3c89daf6b0f9f7da75e8bee93e9405a67#diff-d6484e2cc8cfb181e8f141cb5e72dd4d64dbb456603bfc8240e0a3bebf83878f) image that is currently used in the image documentation.

### To Reproduce

Checkout the `373-tutorial-for-image-processing` branch, first commit (a3c1b1d3c89daf6b0f9f7da75e8bee93e9405a67) and run the 10th cell. (Or run everything, either in the notebook or by serving the documentation with `poetry run mkdocs serve`).

### Expected behavior

If possible, the `invert_colors` method should never fail.

[Stackoverflow](https://stackoverflow.com/questions/41421033/python-invert-binary-image-invert-fails) has some tips on how to achieve that with `ImageOps` and non-RGB formats.

For cases where it is not possible to find a workaround, the internal error from `ImageOps` should be catched and a more user-friendly error should be thrown.

**Comment by zzril** [comment]
This issue currently has a PR (#422), but no assignee. This is intentional.
The PR was created because the commit that fixes the error mentioned in this issue was needed elsewhere already.

Feel free to take over that PR.

**Comment by zzril** [comment]
The bug has been fixed in #419. Images are now converted to RGB format before inverting them.

This might mean that inverting an image twice loses some quality compared to the original image. But we do not seem to test for this, and it's in general a pretty common thing for image operations (example: resize to a smaller size, then back to original).

After having a closer look into the documentation, I also don't see any internal exceptions that could be raised when calling the `PIL`'s `convert` method, so there's nothing more to do here.
